packages/firestore/test/util/test_platform.ts

```diff
@@ -45,7 +45,8 @@ export class FakeWindow {
     });
     this.fakeIndexedDb =
       fakeIndexedDb ||
-      (typeof window !== 'undefined' && window.indexedDB) || null;
+      (typeof window !== 'undefined' && window.indexedDB) ||
```

Comment:
This file got submitted without linter changes, so this is just the result of running prettier.